core/src/main/java/com/datastax/oss/driver/internal/core/context/DefaultDriverContext.java

```diff
@@ -165,8 +167,12 @@
       new LazyReference<>("eventBus", this::buildEventBus, cycleDetector);
   private final LazyReference<Compressor<ByteBuf>> compressorRef =
       new LazyReference<>("compressor", this::buildCompressor, cycleDetector);
+  private final LazyReference<PrimitiveCodec<ByteBuf>> primitiveCodecRef =
+      new LazyReference<>("primitiveCodec", this::buildPrimitiveCodec, cycleDetector);
```

Comment:
`primitiveCodec` was not build before?

Reply:
It was used only in one place, so we were building it directly in `buildFrameCodec`. See the diff on that method.